data/schema/SSVC_Provision_v2.schema.json

```diff
@@ -0,0 +1,119 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://democert.org/ssvc/SSVC_Provision_v2.schema.json",
+    "title": "Decision tree schema definition for SSVC v2",
+    "description": "This provides a schema for a decision tree used to compute SSVC score for a vulnerability",
+    "definitions": {
+        "color_regex": {
```

Comment:
What function does the color regex serve? This seems like it belongs in a UI, not in a data schema. Maybe I'm missing some context?

Reply:
I should also mention that I don't think this color regex was _added_ by this pull request, it appears to have been there previously. This is just where I noticed it. I don't have any problems with the intended changes otherwise, so if you'd prefer to merge this one and deal with the color regex question separately (e.g., in a separate issue), that's fine with me.

Reply:
Allen,
The color regex is for UI - specifically useful in my UI provided to CISA https://democert.org/sssvc/   - surely.  It is an optional field and does not need to be there. It is convenient for the UI that I developed to communicate that's all.

I will leave it there and when we come back we can clean it up. If you think the stretch is much to include UI elements for full description of SSVC decision tree.

Vijay